src/workloads/execution/GroupSpillToDisk.yml

```diff
@@ -0,0 +1,126 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Runs $group queries which are designed to require spilling to disk.
+Keywords:
+- Loader
+- CrudActor
+- QuiesceActor
+- insert
+- aggregate
+- group
+- spill
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 2_000_000
+  FirstFieldUniqueValCount: &FirstFieldUniqueValCount 20
+  RandomStrLength: &RandomStrLength 60
+  MaxPhases: &MaxPhases 5
+
+Actors:
+# Phase 0: Insert documents into the collection.
```

Comment:
[optional] recommend prepending a phase that drops the collection; this is useful for local testing: https://github.com/mongodb/genny/blob/58901acf3ed1a4dd41f55f41f1d46689ff804e51/src/workloads/execution/LookupSBEPushdown.yml#L234